src/java/org/apache/cassandra/utils/memory/MemtableCleanerThread.java

```diff
@@ -27,54 +33,82 @@
  */
 public class MemtableCleanerThread<P extends MemtablePool> extends InfiniteLoopExecutor
 {
-    private static class Clean<P extends MemtablePool> implements InterruptibleRunnable
+    private static final Logger logger = LoggerFactory.getLogger(MemtableCleanerThread.class);
```

Comment:
Nit: missed blank line
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(MemtableCleanerThread.class);

```